### ISSUE TITLE
Subscribe to a topic with a decorator

### ISSUE THREAD

**Issue by glujan** [Mulugruntz/celery-pubsub#235]
Hi @Mulugruntz , I found your package when I was trying to solve the very same issue in one of my projects. Looks all good but I have a small feature request: being able to subscribe to a topic using a decorator, ie:

```
import celery
import celery_pubsub

@celery_pubsub.subscribe("some.topic")
@celery.task
def my_task():
    ...
```

I can prepare a PR for that, the question is if you are open for such an addition to your project?

**Comment by Mulugruntz** [comment]
Hi @glujan !

Thanks for proposing this feature. Actually, it looks super nice! It's funny I didn't think of that :-).

Yes, please make a first PR proposal :-)

**Comment by Mulugruntz** [comment]
Closing, as implemented by #252 🚀
